sys/socket_bluetooth.txt

```diff
@@ -22,39 +22,39 @@ resource sock_bt_hci[sock_bt]
 socket$bt_hci(fam const[AF_BLUETOOTH], type const[SOCK_RAW], proto const[BTPROTO_HCI]) sock_bt_hci
 bind$bt_hci(fd sock_bt_hci, addr ptr[in, sockaddr_hci], addrlen len[addr])
 ioctl$sock_bt_hci(fd sock_bt_hci, cmd flags[bt_hci_ioctl], arg buffer[inout])
-setsockopt$HCI_DATA_DIR(fd sock_bt_hci, level const[0], opt const[HCI_DATA_DIR], arg ptr[in, int32], arglen len[arg])
-setsockopt$HCI_TIME_STAMP(fd sock_bt_hci, level const[0], opt const[HCI_TIME_STAMP], arg ptr[in, int32], arglen len[arg])
-setsockopt$HCI_FILTER(fd sock_bt_hci, level const[0], opt const[HCI_FILTER], arg ptr[in, hci_ufilter], arglen len[arg])
+setsockopt$bt_hci_HCI_DATA_DIR(fd sock_bt_hci, level const[0], opt const[HCI_DATA_DIR], arg ptr[in, int32], arglen len[arg])
```

Comment:
The idea is to be able to enable or disable entire some socket family related syscalls via regexps like `ioctl$sock_inet*`, `setsockopt$inet_dccp*`, etc.

For consistency I decided to name all `s/getsockopt` and `ioctl` calls as `setsockopt$typename_OPTIONNAME` and `ioctl$sock_typename_CMDNAME`.